database/similarProperties.js

```diff
@@ -0,0 +1,19 @@
+const mongoose = require('mongoose');
+const db = require('./index.js');
+mongoose.Promise = global.Promise;
+
+const similarPropertiesSchema = new mongoose.Schema({
+  propertyID: Number,
```

Comment:
FYI in my documents this is stored as `listingId` and is a 4-digit number starting with `1001`